### PR TITLE
OCPBUGS-27186: Ns ipsec 4.15

### PR DESCRIPTION
**Pull request by yuvalk**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -157,3 +157,5 @@ require (
 	k8s.io/kms v0.28.4 // indirect
 	k8s.io/kube-aggregator v0.28.2 // indirect
 )
+
+replace github.com/openshift/api => ../api/
```
Comment (yuvalk) [inline go.mod:161]: this is actually
https://github.com/yuvalk/api/tree/cherry-pick-1667-to-release-4.15

**Comment by JoshSalomon** [comment]
retest

**Comment by yuvalk** [comment]
/hold
this need to wait first for https://github.com/openshift/api/pull/1719 to merge

**Comment by yuvalk** [comment]
/hold cancel
the api pr merged

**Comment by yuvalk** [comment]
@jcaamano according to tide, need also the `cherry-pick-approved` label
can you please add that?

**Pull request merged**